ethereum/core/src/test/java/org/hyperledger/besu/ethereum/privacy/PrivateTransactionValidatorTest.java

```diff
@@ -74,7 +74,17 @@ public void transactionWithInvalidChainIdShouldReturnWrongChainId() {
     validator = new PrivateTransactionValidator(Optional.of(BigInteger.ONE));
 
     ValidationResult<TransactionInvalidReason> validationResult =
-        validator.validate(privateTransactionWithChainId(999), 0L);
+        validator.validate(privateTransactionWithChainId(Optional.of(BigInteger.TWO)), 0L);
+
+    assertThat(validationResult).isEqualTo(ValidationResult.invalid(WRONG_CHAIN_ID));
+  }
+
+  @Test
+  public void transactionWithoutChainIdWithValidatorUsingChainIdShouldReturnWrongChainId() {
```

Comment:
isn't this valid? I.e. if the transactions doesn't have a chainID, but the Validator does, then that's ok isn't it?